shm/Ship.cpp

```diff
@@ -0,0 +1,36 @@
+#include <string>
+
+class Ship {
+public:
```

Comment:
Why you have this in .cpp file?

Reply:
In the begining I didn't know to properly split things between c++ and header file, but I belive I do know now ;p